Source/ZoomNet/Resources/Groups.cs

```diff
@@ -1,6 +1,44 @@
+using Pathoschild.Http.Client;
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace ZoomNet.Resources;
 
-public class Groups
+/// <summary>
+/// Allows you to manage Groups.
+/// </summary>
+public class Groups : IGroups
 {
-	
+	private readonly IClient _client;
+
+	internal Groups(IClient client)
+	{
+		_client = client;
+	}
+
+	/// <summary>
+	/// Adds a user to a group.
+	/// </summary>
+	/// <param name="groupId">The ID of the group.</param>
+	/// <param name="userEmail">The email of the user to add to the group.</param>
+	/// <param name="cancellationToken">The cancellation token.</param>
+	/// <returns>A task representing the operation. The result will be true if the user was added successfully, false otherwise.</returns>
+	/// <inheritdoc/>
+	public async Task<bool> AddUserToGroupAsync(string groupId, string userEmail, CancellationToken cancellationToken = default)
```

Comment:
Two observations:

1. The API allows multiple users to be added to a group with a single call. Rather than accepting a single user email, this method should accept an enumeration of email addresses.
2. The API returns the ID of each created group members. Rather than returning a bool, this method should return an array of strings.

What I have in mind is something like this:
```csharp
public async Task<string[]> AddUsersToGroupAsync(string groupId, IEnumerable<string> emails, CancellationToken cancellationToken = default)
```

When this method is in place, we can add a convenient extension method to `Extensions\Public.cs` to add a single user to a group. Something like this: 

```csharp
public static async Task<string> AddUserToGroupAsync(this IGroups groupsResource, string groupId, string emailAddress, CancellationToken cancellationToken = default)
{
	var result = await groupsResource.AddUsersToGroupAsync(groupId, new[] { emailAddress }, cancellationToken).ConfigureAwait(false);

	// We added a single member to a group therefore the array returned from the Zoom API contains a single element
	return result.Single();
}
```

Reply:
Don't forget the extension method in `Extensions\Public.cs` to add a single user to a group.